FF1Lib/FF1Rom.cs

```diff
@@ -102,7 +103,23 @@ public static async Task<FF1Rom> CreateAsync(Stream readStream)
 
 			return rom;
 		}
+		public class RandomizerDigest
+		{
+			//This JSON gets embeded into the rom, so the property values have been
+			//shortened to save space
+			public String s { get; set; } //Randomizer Seed
+			public String f { get; set; } // Encoded flag string
+			public String g { get; set; } // Git SHA used to generate this seed
+			public String r { get; set; } // SHA-1 of the input rom
 
+			public RandomizerDigest(String _seed, String _flags, String _gitSha, String _romSha)
```

Comment:
This constructor isn't necessary AND since all the types are identical it can be error prone. If somebody gets the order wrong it blows up! I highly recommend reading this article on parameterless constructors. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/how-to-initialize-objects-by-using-an-object-initializer

Reply:
I had no idea this existed! it makes sense.
Done